xdem/coreg.py

```diff
@@ -705,6 +705,10 @@ def residuals(self, reference_dem: Union[np.ndarray, np.ma.masked_array],
         # Calculate the DEM difference
         diff = ref_arr - aligned_dem
 
+        # Sometimes, the float minimum (for float32 = -3.4028235e+38) is returned. This and inf should be excluded.
```

Comment:
how come this extreme float value is returned in the first place?
should we not fix it there instead?

Reply:
Added #193!